From 51cd4cde58d46d76aab4d09a12aa3475e641aabf Mon Sep 17 00:00:00 2001
From: Alexandru Ardelean <alexandru.ardelean@analog.com>
Date: Mon, 9 Dec 2019 15:07:00 +0200
Subject: [PATCH 10/12] drm: adv7511: add includes for struct types

In upstream Linux, the DRM subsystem has organized their includes. In
order for this file to build, these includes are required so that the
compiler can evaluate these structs.

Signed-off-by: Alexandru Ardelean <alexandru.ardelean@analog.com>
(cherry picked from commit d8b7246b3bbc961283088121dd23887a7581c7ad)
---
 drivers/gpu/drm/bridge/adv7511/adv7511_debugfs.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/bridge/adv7511/adv7511_debugfs.c b/drivers/gpu/drm/bridge/adv7511/adv7511_debugfs.c
index 86bc77a7e2e1..90735d51b423 100644
--- a/drivers/gpu/drm/bridge/adv7511/adv7511_debugfs.c
+++ b/drivers/gpu/drm/bridge/adv7511/adv7511_debugfs.c
@@ -8,6 +8,8 @@
 #include <linux/types.h>
 #include <linux/debugfs.h>
 #include <drm/drm_print.h>
+#include <drm/drm_device.h>
+#include <drm/drm_file.h>
 
 #include "adv7511.h"
 
-- 
2.17.1

